.env_SAMPLE

```diff
@@ -108,9 +122,9 @@ export WORDPRESS=http://www.consumerfinance.gov
 # TESTING (optional) - for handling testing of the site.
 ########################################################
 
-#export HTTP_HOST=localhost
-#export HTTP_PORT=8000
-#export SELENIUM_URL=http://localhost:4444/wd/hub
+#export SITE_DESC=cfgov
+#export TEST_HTTP_HOST=localhost
+#export TEST_HTTP_PORT=8000
```

Comment:
Should this whole `TESTING` section be removed since you're resetting these three variables below in the `Project configuration` section?